ldap/src/main/java/org/springframework/security/ldap/server/ApacheDSContainer.java

```diff
@@ -186,6 +186,10 @@ public void setPort(int port) {
 		this.port = port;
 	}
 
+	public int getPort() {
```

Comment:
Would you please put this into a separate commit and add a new ticket so that developers are alerted separately to the new feature? Thank you!

Reply:
done